app/controllers/v0/virtual_agent/user_controller.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module V0
+  module VirtualAgent
+    class UserController < SignIn::ServiceAccountApplicationController
```

Comment:
should this be `UsersController`?

Reply:
ya, do you think there's a better name or is `UsersController` fine?